isis/src/lro/apps/lronaccal/lronaccal.xml

```diff
@@ -46,6 +45,33 @@
     image pixels.
     </p>
 
+    <p>
+      The DN level in an uncalibrated image is the sum of the true signal from the scene, 
+      the bias, the dark current, and random noise in all 3 components. The random noise in
+      the true signal and dark current is called shot noise and the random noise in the bias
+      is called read noise. The true signal, bias, and dark current are defined as mean
+      values so that if the random noise were averaged down to insignificance by taking a 
+      very large number of images and averaging them, the resulting image would be the true 
+      scene, bias, and dark current with no systematic error. That implies the statistical 
+      distribution of the random noise has an average of zero, and therefore the random noise
+      has both positive and negative values, except for the trivial case of zero random noise.
+      The random noise is both additive and subtractive to the signal by definition.
+    </p>
+
+    <p>
+      Let's look at the case of a calibrated image for which the true signal is zero, a dark 
+      image. In calibration, the mean bias and dark current are subtracted, but the random 
+      noise cannot be subtracted, because only the statistical distribution of the random 
+      noise can be known in advance, not the specific value for that pixel for that image. 
+      Because the random noise values are both positive and negative, when the true signal 
+      is zero the calibrated image should have both positive and negative values. If it had 
+      only positive values, the mean would be positive and therefore it would have a 
+      systematic error, since the true signal is zero. It is not possible to eliminate the 
+      random noise entirely, but it is defined as both additive and subtractive to the signal 
+      to minimize the systematic error in the calibrated image. That forces calibrated dark 
+      images to have both positive and negative values.
```

Comment:
I am concerned that this paragraph is not very clear. From what I am reading, the random noise is additive, with a mean of zero (and some variance, σ^2). Since the variance is not equal to zero, a sample taken from what sounds like a normal distribution can have a negative value. Is that what the above is saying?

Then the mean bias (MB) and dark current (DC) are additive as well, though they can have negative values? So, if the DN = 0, should we see:

DB + MB + DC = N(μ, σ^2), where N(μ, σ^2) is the error as sampled from a normal distribution.

Right now, the additive vs. subtractive seems to muddy the conversation. It seems that all the distinction does is flip the sign on the error term.

Reply:
> I am concerned that this paragraph is not very clear. From what I am reading, the random noise is additive, with a mean of zero (and some variance, σ^2). Since the variance is not equal to zero, a sample taken from what sounds like a normal distribution can have a negative value. Is that what the above is saying?

Yes, that is what it is saying. Is it the last sentence of the first paragraph that is confusing? I believe Dave is restating the previous sentence in a different way. Trying to emphasize that zero +- the noise results in negative output when the noise is negative. Not that it makes a difference, but the noise is modeled as a poisson distribution.
> 
> Then the mean bias (MB) and dark current (DC) are additive as well, though they can have negative values? So, if the DN = 0, should we see:
> 
> DB + MB + DC = N(μ, σ^2), where N(μ, σ^2) is the error as sampled from a normal distribution.
> 
> Right now, the additive vs. subtractive seems to muddy the conversation. It seems that all the distinction does is flip the sign on the error term.

Reply:
It is these sentences on combination:

`The random noise is both additive and subtractive to the signal by definition.`
and
` In calibration, the mean bias and dark current are subtracted, but the random 
       noise cannot be subtracted, because only the statistical distribution of the random 
       noise can be known in advance, not the specific value for that pixel for that image.`

The idea that the values are added and subtracted does not make sense. I would guess that the values are added and that some of them can be negative. Can we just say that? If not, what are the rules are used to determine whether the sampled random error is added or subtracted. 

It also does not make sense that the 'the random noise cannot be subtracted' because the statistical distribution can be known. These two are completely orthogonal to one another as far as I know. Some value is sampled from a distribution. Literally anything could be done with that value.